### PR TITLE
bugs fixed on grid properties, props validation, and myProvider.

### LINKED ISSUE #74: Warning: Failed prop type: The prop `xs` of `Grid` can only be used together with the `item` prop.
image

On:
image

### LINKED ISSUE #75: Warning: Failed prop type: Invalid prop `item.id` of type `number` supplied to `ProductItem`, expected `object`.
image

At ProductItem component.

### LINKED ISSUE #76: Warning: Failed prop type: Invalid prop `children.$$typeof` of type `symbol` supplied to `MyProvider`, expected `object`.
image

At MyProvider.

### PR DESCRIPTION
**Pull request by santiagoejm**

### CODE REVIEW

**Comment by santiagoejm** [comment]
Not sure, if the solution of the bug on MyProvider is the most elegant, there are no more errors displayed in the console, but still appears to be 2 unsolved issues, which I can not  access inspecting the browser.

Let me know, what do you think

File: frontend/.env.development [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_ADMIN_ROUTE=http://localhost:3030/
\ No newline at end of file
+REACT_APP_ADMIN_ROUTE=http://localhost:3030/
+REACT_APP_SERVER_URL=http://localhost:5000
```
Comment (glaubernespoli) [inline frontend/.env.development:2]: do the merge; you should get a conflict here

Comment (glaubernespoli) [reply frontend/.env.development:2]: (maybe)

File: frontend/src/containers/Home/product-list/ProductItem.jsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ ProductItem.defaultProps = {
 };
 
 ProductItem.propTypes = {
-  item: PropTypes.objectOf(PropTypes.object)
+  item: PropTypes.objectOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string]))
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/product-list/ProductItem.jsx]: this should be string only. when we start taking the info from the backend we'll have strings as ids, not number. reason you're getting numbers also is probably due to the static file having different types for ids

**Review by glaubernespoli — CHANGES_REQUESTED** [review]
Check requested changes;
also do pull of the main to your local branch to solve conflicts before the pull request

**Review by glaubernespoli — APPROVED** [review]

**Pull request merged**